README.md

```diff
@@ -49,3 +49,33 @@ These can be enabled when running this driver through Appium, via the `--allow-i
 |Feature Name|Description|
 |------------|-----------|
 |`system_shell`|Allows to execute shell scripts on the machine. Read [appium-mac-driver#38](https://github.com/appium/appium-mac-driver/pull/38) for more details on the implementation |
+
+## Desired Capabilities
+
+Should be same for [Appium](https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/caps.md)
+
+Differences are noted here:
+
+### Handling Appium for mac
+
+
+|Capability|Description|Values|
+|----------|-----------|------|
+| `AppiumForMac` | Specify the host name to the app for mac application. Defaults to `127.0.0.1` | e.g., `localhost` |
+| `a4mPort` | Specify the port to the app for mac application. Defaults to `4622` | e.g, `4622`, `8080` |
+| `a4mAppPath` | Specify the path to the app for mac application. It helps to launch `AppiumForMac` application in a custom path. Defaults to `/Applications/AppiumForMac.app` | e.g, `/Applications/CustomAppiumForMac.app` |
```

Comment:
It would be smarter to find the app by its bundle id using searchd automatically if this cap is not set: https://apple.stackexchange.com/questions/115947/locating-an-app-by-its-bundle-identifier-from-the-command-line